test/integration_test/common_test.go

```diff
@@ -140,6 +161,11 @@ var storkVersionCheck bool
 var cloudDeletionValidate bool
 var isInternalLBAws bool
 var pxNamespace string
+var storkVersion string
+var testrailHostname string
+var testrailUsername string
```

Comment:
nit.HostName, UserName

Reply:
This has been named so, because we want to indentify that these are the host and username for testrail specifically.